etc/di.xml

```diff
@@ -62,6 +62,9 @@
 
     <!-- For debug endpoint -->
     <preference for="Bolt\Boltpay\Api\DebugInterface" type="Bolt\Boltpay\Model\Api\Debug" />
+
+    <!-- For debug save endpoint -->
+    <preference for="Bolt\Boltpay\Api\DebugSaveInterface" type="Bolt\Boltpay\Model\Api\DebugSave" />
```

Comment:
We want to update settings so maybe we can call this API Bolt\Boltpay\Model\Api\UpdateSettings